### PR TITLE
Vedtektsendringer 11. Mars 2019

### PR DESCRIPTION
**Pull request by s0lvang**
Endringene som ble vedtatt av generalforsamlingen den 11. Mars 2019

### CODE REVIEW

File: paragrafer/navn.tex [hunk 1]
```diff
@@ -18,7 +18,7 @@ \section{Grupperinger i linjeforeningen}
 
 \subsection{Kjernekomité}
 
-I Online er en kjernekomité en komité der komiteens leder er en del av Hovedstyret. Seniorkomiteen er også en kjernekomité. 
+I Online er en kjernekomité en komité som er beskrevet i 4.2
```
Comment (oleast) [inline paragrafer/navn.tex]: Can this be linked automatically to the paragraph?

File: paragrafer/strukturLedelseOgOrganisasjon.tex [hunk 2]
```diff
@@ -99,8 +99,7 @@ \subsection{Seniorkomiteen}{
 %----
 \section{Nodekomiteer}
 \vspace{23pt}
-En nodekomite er underlagt en av komiteene beskrevet i 4.2, eller direkte underlagt Hovedstyret, og plikter å holde den ansvarlige kjernekomiteen løpende oppdatert på sitt virke. Nodekomiteer som har eksistert i to år eller mer kan bare offisielt nedlegges av Generalforsamlingen.
-
+En nodekomite er underlagt en av komiteene beskrevet i 4.2, eller direkte underlagt Hovedstyret, og plikter å holde den ansvarlige kjernekomiteen løpende oppdatert på sitt virke.
```
Comment (oleast) [inline paragrafer/strukturLedelseOgOrganisasjon.tex]: Can this be linked automatically to the paragraph?

**Review by oleast — APPROVED** [review]
Looks good :smile:

**Comment by sklirg** [comment]
Did you rebuild the PDF as well? Cant see it in the diff

**Comment by sklirg** [comment]
Oops nvm saw it now. Wasn't in the original. 🙈

**Pull request merged**